### PR TITLE
(feat): Added Latest Blogs Component

### LINKED ISSUE #167: Feature Request: Add a Featured Blog/News Section
### Is your feature request related to a problem? Please describe.

So, as we have a few blogs on the Uptane site, we want to feature the three most recent ones on the main page of the site.

### Describe how you would like this problem to be resolved

Create a component as displayed below in the Figma design:

image

This component should display the most recent blogs on the site along with their images and titles."

### Describe alternatives you've considered

_No response_

### Additional context.

_No response_

### PR DESCRIPTION
**Pull request by Abhijay007**
# Summary of Changes

Added Latest Blogs Component

## Related Issue

Closes #167 

## Checklist

- [x] I have read and followed the [project's contribution guidelines](https://github.com/secure-systems-lab/lab-guidelines/blob/master/dev-workflow.md), including code style and commit message conventions.
- [x] My code is well-documented, and I've updated relevant documentation.
- [x] I have tested these changes on my local environment.
- [x] All tests pass, and there are no new linting errors.
- [x] I have reviewed and proofread my code and the changes.
- [x] The branch is up-to-date with the base branch.

## Screenshots (if applicable)

image

## Additional Context

in future we need a workflow to automate the latest blogs on the home page

### CODE REVIEW

File: static/data/latestBlogs/data.js
Comment (hexsecs) [inline static/data/latestBlogs/data.js:1]: @Abhijay007 : How does this file get updated? Do we need to create an automation? or will we need to update this every time a new blog is released?

**Review by hexsecs — COMMENTED** [review]
Looks good, just one open question about the static section.

**Review by jhdalek55 — APPROVED** [review]
I was requested to review this. It looks fine to me, but remember I don't write code.

File: static/data/latestBlogs/data.js
Comment (Abhijay007) [reply static/data/latestBlogs/data.js:1]: @hexsecs For now we can update this manually, but will try to automate this using a workflow, I will create a separate PR for that

**Pull request merged**